src/main/java/seedu/address/model/student/Note.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.student;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a note for a Student in WellNUS
+ */
+public class Note {
```

Comment:
Maybe can impose some limit on the length of `Note`, can take reference from `Description` class.